mne_bids/utils.py

```diff
@@ -21,6 +21,21 @@
 from .io import _parse_ext
 
 
+def print_dir_tree(dir):
+    """Recursively print a directory tree starting from `dir` [1].
```

Comment:
Should I add a test for this? It'd probably be a very trivial test, but for coverage reasons?

Reply:
sure, +1 for that